### DISCUSSION TITLE
Block applications

### DISCUSSION THREAD

**Discussion by fayddelight** [SelfControlApp/selfcontrol#876] [Ideas]
I love this app and have been using it extensively.

It would be a great feature if applications like WhatsApp for example could be blocked as well.

Edit: Similar idea was posted in 2022 but did not get any responses hence I decided to open this for discussion in ideas.

**Comment by desmond-dsouza** [comment]
+1
I managed to block WhatsApp by putting a few different WhatsApp and FB domains on the list. Then removed them from the list as I wanted WA access while blocking others. Forgot to save that list. Now cannot seem to replicate, either don't recall them or domains have moved.
